core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -143,73 +143,6 @@ class KafkaConfigTest {
 
   }
 
-  @Test
-  def testAdvertiseDefaults(): Unit = {
-    val port = "9999"
-    val hostName = "fake-host"
-
-    val props = TestUtils.createBrokerConfig(0, TestUtils.MockZkConnect)
-    props.remove(KafkaConfig.ListenersProp)
-    props.put(KafkaConfig.HostNameProp, hostName)
-    props.put(KafkaConfig.PortProp, port)
-    val serverConfig = KafkaConfig.fromProps(props)
-    val endpoints = serverConfig.advertisedListeners
-    val endpoint = endpoints.find(_.securityProtocol == SecurityProtocol.PLAINTEXT).get
-    assertEquals(endpoint.host, hostName)
-    assertEquals(endpoint.port, port.toInt)
-  }
-
-  @Test
-  def testAdvertiseConfigured(): Unit = {
```

Comment:
I saw you removed all the `AdvertiseListener`'s tests here. Should we add or keep and update some tests for it?